source/index.html.md

```diff
@@ -168,14 +197,14 @@ patient/Patient.read | Permission to read Patient resource for the current patie
 patient/Observation.read | Permission to read Observation resource for the current patient.
 openid, profile | Permission to retrieve information about the current logged-in user. Required for EHR launch.
 launch | Permission to obtain launch context when app is launched from an EHR. Required for EHR launch.
-launch/patient | When launching outside the EHR, ask for a patient to be selected at launch time. Required for stand-alone launch.
+launch/patient | When launching outside the EHR, ask for a patient to be selected at launch time. Required ONLY for patient standalone launch.  See this section: [Standalone App Launch for Patient Access Workflow](#standalone-app-launch-for-patient-access-workflow).
```

Comment:
Slight clarification: It's only required if you want a patient selected when doing a patient facing standalone launch. Apps can choose to use user/* scopes and wouldn't need a launch at all.